Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
```

Comment:
Similar to the root password, the user password is hardcoded. It's advisable to use a more secure method to handle passwords, such as Docker secrets or environment variables set at runtime.